radix-engine/src/system/system.rs

```diff
@@ -2869,99 +2884,7 @@ where
     feature = "std",
     catch_unwind(crate::utils::catch_unwind_system_panic_transformer)
 )]
-impl<'a, Y, V> ClientCryptoUtilsApi<RuntimeError> for SystemService<'a, Y, V>
-where
-    Y: KernelApi<System<V>>,
-    V: SystemCallbackObject,
-{
-    #[trace_resources(log=message.len())]
```

Comment:
`trace_resource` macro should be moved to `verify_bls12381_v1()` function.

Reply:
good point!